lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_georgia.govspeak.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for :title do %>
+  Marriage in Georgia
+<% end %>
+
+<% content_for :body do %>
+  Contact the [Ministry of Justice of Georgia](http://www.sda.gov.ge/?page_id=7303&lang=en) to find out about local marriage laws, including what documents you’ll need.
+
+  <% if calculator.ceremony_country == 'georgia' && calculator.resident_of_ceremony_country? %>
+
+    ^You should [get legal advice](/government/collections/list-of-lawyers) before making any plans.
+
+  <% else %>
+
+    ^You should [get legal advice](/government/collections/list-of-lawyers) and check the [travel advice for Georgia](/foreign-travel-advice/georgia) before making any plans.^
+
+  <% end %>
```

Comment:
Can you change this `if` block to:

```erb
<% if calculator.resident_of_ceremony_country? %>
  <%= render partial: 'get_legal_advice.govspeak.erb' %>
<% else %>
  <%= render partial: 'get_legal_and_travel_advice.govspeak.erb',
             locals: { calculator: calculator } %>
<% end %>
```

We should be trying to reuse existing partials as much as possible to help us avoid introducing more duplication. When I'm making this sort of change, I'll search the project for similar text (e.g. search for "[get legal advice]") to see whether the text has already been extracted to a partial.